Deck_Shuffler/Deck_Shuffler/Card.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    public class Card
```

Comment:
Ooh. Nice immutable class.

Reply:
Are you referring to the properties I assigned for the variables? {get;}